pkg/app/piped/cloudprovider/terraform/terraform.go

```diff
@@ -58,10 +88,13 @@ func (t *Terraform) Init(ctx context.Context, w io.Writer) error {
 	args := []string{
 		"init",
 	}
-	for _, v := range t.vars {
+	if t.options.noColor {
+		args = append(args, "-no-color")
+	}
+	for _, v := range t.options.vars {
 		args = append(args, fmt.Sprintf("-var=%s", v))
 	}
-	for _, f := range t.varFiles {
+	for _, f := range t.options.varFiles {
 		args = append(args, fmt.Sprintf("-var-file=%s", f))
 	}
```

Comment:
nits, should we extract it as `t.makeCommandArgs` private method 🤔

Reply:
Sure. Applied that idea. 👍